.gitignore

```diff
@@ -6,9 +6,7 @@ node_modules
 .DS_Store
 
 # Builds
-bin
+bin/
 roo-cline-*.vsix
 
-# Local prompts
-prompts
 .clinerules
```

Comment:
I think we might want to remove this as well so we can create a .clinerules for working on Cline